domino/flyte/task.py

```diff
@@ -149,6 +165,8 @@ def _resolve_job_properties(self, domino_job_config: DominoJobConfig) -> Dict[st
             resolved_job_config["apiKey"] = domino_job_config.ApiKey
             resolved_job_config["command"] = domino_job_config.Command
             resolved_job_config["title"] = domino_job_config.Title
+            resolved_job_config["inputInterfaceBase64"] = None
+            resolved_job_config["inputOutputInterfaceBase64"] = None
```

Comment:
These lines are basically noops

Reply:
sure but i think explicit is almost always better than implicit. but i guess since im taking the args out of the domino config object i can take this out